t/18-job-constants.t

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env perl -w
+
+# Copyright (C) 2014-2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```

Comment:
Please change according to https://github.com/os-autoinst/openQA/pull/2795/files if you agree with https://github.com/os-autoinst/openQA/pull/2795